### PR TITLE
usm: postgres: tests: Do not use kafka test context

### PR DESCRIPTION
**Pull request by guyarb**
### What does this PR do?

Declares a dedicated test context for postgres

### Motivation

Not using the same struct of kafka

### CODE REVIEW

File: pkg/network/usm/postgres_monitor_test.go [hunk 1]
```diff
@@ -59,18 +59,32 @@ func generateSelectLimitQuery(limit int) string {
 	return fmt.Sprintf("SELECT * FROM dummy limit %d", limit)
 }
 
+// testContext shares the context of a given test.
```
Comment (amitslavin) [inline pkg/network/usm/postgres_monitor_test.go:62]: fix testContext

File: pkg/network/usm/postgres_monitor_test.go [hunk 2]
```diff
@@ -59,18 +59,32 @@ func generateSelectLimitQuery(limit int) string {
...
+// It contains common variable used by all tests, and allows extending the context dynamically by setting more
+// attributes to the `extras` map.
+type pgTestContext struct {
+	// The address of the server to listen on.
+	serverAddress string
```
Comment (amitslavin) [inline pkg/network/usm/postgres_monitor_test.go:67]: nit: can be changed to postgresPort as being used in the tests

**Review by amitslavin — CHANGES_REQUESTED** [review]

File: pkg/network/usm/postgres_monitor_test.go [hunk 3]
```diff
@@ -59,18 +59,32 @@ func generateSelectLimitQuery(limit int) string {
...
+	// The port to listen on.
+	serverPort string
```
Comment (amitslavin) [inline pkg/network/usm/postgres_monitor_test.go:69]: nit: can be changed to postgresPort as being used in the tests

Comment (guyarb) [reply pkg/network/usm/postgres_monitor_test.go:69]: I think it should be clear enough that's postgres (we're in postgres_monitor_test, and the struct is `pgTest...`

**Review by amitslavin — APPROVED** [review]

**Pull request merged**